$NetBSD: patch-configure,v 1.2 2015/12/29 23:34:54 dholland Exp $

Don't explicitly link libgcc or libstdc++. Fixes build with clang.

--- configure.orig	2014-05-09 10:11:45.000000000 +0000
+++ configure
@@ -17107,7 +17107,7 @@ if test "x$ac_cv_lib_gcc_main" = xyes; t
 #define HAVE_LIBGCC 1
 _ACEOF
 
-  LIBS="-lgcc $LIBS"
+#  LIBS="-lgcc $LIBS"
 
 fi
 
@@ -17146,7 +17146,7 @@ if test "x$ac_cv_lib_gcc_s_main" = xyes;
 #define HAVE_LIBGCC_S 1
 _ACEOF
 
-  LIBS="-lgcc_s $LIBS"
+#  LIBS="-lgcc_s $LIBS"
 
 fi
 
@@ -17185,7 +17185,7 @@ if test "x$ac_cv_lib_stdcpp_main" = xyes
 #define HAVE_LIBSTDC__ 1
 _ACEOF
 
-  LIBS="-lstdc++ $LIBS"
+#  LIBS="-lstdc++ $LIBS"
 
 fi
 
